.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
-
+        condition: $TRAVIS_OS_NAME = 'linux' && $CC = 'gcc'
```

Comment:
Really? that's messed up. I sometimes don't understand what is Travis trying to do...